src/components/Chat/ChatMessage.tsx

```diff
@@ -198,29 +194,26 @@ class ChatMessage extends UIComponent<Extendable<ChatMessageProps>, any> {
     })
 
     const contentElement = Slot.create(content, {
-      styles: styles.content,
-      variables: variables.content,
+      defaultProps: {
+        styles: styles.content,
+        variables: variables.content,
+      },
       render: renderContent,
     })
 
     return (
-      <Layout
-        start={!mine && avatarElement}
-        main={
-          <Layout
-            className={classes.content}
-            vertical
-            start={
-              <>
-                {!mine && authorElement}
-                {timestampElement}
-              </>
-            }
-            main={contentElement}
-          />
-        }
-        end={mine && avatarElement}
-      />
+      <>
+        {!mine && avatarElement}
+        <Slot
+          key="chat-message-body"
```

Comment:
As I remember, we don't need keys inside fragments

Reply:
indeed, this was a leftover from when I was fixing the console errors, I thought it's coming from here; thanks, good catch